README.md

```diff
@@ -152,6 +152,94 @@ async function search() {
 search().catch(console.log);
 ```
 
+### With AWS SigV4 signing
+```javascript
+const endpoint = ""; // OpenSearch domain URL e.g. https://search-xxx.region.es.amazonaws.com
+const { Client } = require('@opensearch-project/opensearch');
+const AwsV4Signer = require('@opensearch-project/opensearch/AwsV4Signer');
```

Comment:
I think I'd want it to match above and be namespaced under `aws`:

```javascript
const { AwsSigV4Signer } = require ('@opensearch-project/opensearch/aws')
```

Reply:
This sounds right. I fixed the namespace to allow this kind of import.